src/elements/content-uploader/Footer.js

```diff
@@ -36,17 +40,33 @@ const Footer = ({ isLoading, hasFiles, errorCode, onCancel, onClose, onUpload, f
         <div className="bcu-footer">
             <div className="bcu-footer-left">
                 {onClose ? (
-                    <Button isDisabled={hasFiles} onClick={onClose} type="button">
+                    <Button
+                        disabled={isCloseButtonDisabled}
```

Comment:
what's the difference between disabled and isDisabled?

Reply:
isDisabled sets the styles/classes for the button, while disabled is passed onto the html element <button />, which prevents focus on the element